code/game/gamemodes/meteor/meteors.dm

```diff
@@ -170,7 +171,8 @@
 	icon_state = "medium_radioactive"
 
 /obj/effect/meteor/radioactive/Bump(atom/a)
-
+	if(loc == null)
```

Comment:
it already inherits, but it inherits at the end of the proc.

Like the one right below this applies radiation to people in range before calling meteor/bump.